xcodeproj/internal/targets.bzl

```diff
@@ -48,8 +48,6 @@ def _is_test_bundle(target, deps):
     if deps == None:
         return False
 
-    # TODO: Once we have a minimum version of Bazel 5.3+, we can check for
-    # `RunEnvironmentInfo` and `*XcTestBundleInfo` instead
```

Comment:
Please put this comment back, since we haven't changed the check yet (though we can now).

Reply:
Can we get the comment back?